docs/docs/adding-search-with-algolia.md

```diff
@@ -40,32 +46,33 @@ module.exports = {
         chunkSize: 10000, // default: 1000
       },
     },
+    `gatsby-plugin-styled-components`,
   ],
 }
 ```
 
 Notice that we're loading `queries` from a file at `./src/utils/algolia.js` (you can of course put it wherever you like) and our Algolia ID and API key from `.env` so let's add those files.
 
-```env:title=.env
+```sh:title=.env
```

Comment:
```suggestion
```text:title=.env
```

Reply:
I usually get better syntax highlighting (e.g. of comments) in `.env` files with `sh` but I assume `text` is convention?